package.json

```diff
@@ -180,12 +190,18 @@
     ],
     "ignore": [
       "tests/*.js",
-      "tools/locale-helper/*.js"
+      "tools/locale-helper/*.js",
+      "*/**/*.js",
+      "*.js"
     ],
     "envs": [
       "node",
       "browser",
       "mocha"
     ]
+  },
+  "dependencies": {
+    "@types/node": "^12.0.8",
+    "@types/requestidlecallback": "^0.1.1"
```

Comment:
@vsvipul not sure what's this?

Reply:
This was added because @vsvipul was getting a error when he migrated a ts that used this function. I will look into this.